_static/api-examples.md

```diff
@@ -152,7 +152,7 @@ You can sign a manifest with either the e-Manifest ID:
 ```bash
 curl -i -X POST -H 'Content-Type: application/json' \
   --data @sign-manifest-emanifestid.json \
-  https://e-manifest.18f.gov/api/0.1/manifest/sign
+  https://e-manifest.18f.gov/api/v1/manifests/:manifest_id/sign
```

Comment:
what do you think about the use of a nested route here? I like it because it is standard for Rails, but the weird thing is that we accept *either* an id or a tracking number there...not sure what the API best practice is for something like this. As you stated earlier, explicitness of a named param is nice for knowing intent, but it's also nice to have nested routes so you know when you are acting on an object.